services/director-v2/src/simcore_service_director_v2/utils/dask_client_utils.py

```diff
@@ -131,7 +131,7 @@ async def _connect_with_gateway_and_create_cluster(
             logger.info("Cluster dashboard available: %s", cluster.dashboard_link)
             # NOTE: we scale to 1 worker as they are global
             await cluster.adapt(active=True)
-            client = await cluster.get_client()  # type: ignore
+            client = await cluster.get_client()
```

Comment:
so you removed all these ignores. how does your Pylance cope with that?

Reply:
looks happy, no complains. Maybe some other changes fixed it?